docs/UserGuide.md

```diff
@@ -189,7 +189,7 @@ Examples:
 
 Sorts the list of persons being viewed by name or date of last visit in ascending or descending order.
 
-Format: `sort PARAMETER/ORDER`
+Format: `sort PARAMETER PREFIX/ORDER`
```

Comment:
I think would be better to use `PARAMETER_PREFIX/ORDER` instead to indicate it is 1 placeholder instead of 2

Reply:
same for the subsequent lines